### PR TITLE
Initial presentation on convolution.

### PR DESCRIPTION
**Pull request by abergeron**

### CODE REVIEW

File: conv/conv.tex [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+\setcounter{page}{1}
+
+\section{Convolution}
+
+% Definitions
+\begin{frame}
+Some vocabulary:
+\begin{description}[feature maps]
+\item[feature map] analogue to layer (the activations, not the weights) for a neural network.
```
Comment (lamblin) [inline conv/conv.tex]: For me, "feature map" was equivalent to "channel". This is why the documentation of `nnet.conv2d` talks about "a stack of feature maps".

File: conv/conv.tex [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+\item[channel] a single 2D "image" which composes a feature map.
+\item[filter] weight (as in layer weights).
```
Comment (lamblin) [inline conv/conv.tex]: One filter is one 2D "image", that will be convolved with one input feature map, to give one component of the output feture map. There are (input_channels \* output_channels) filters.

File: conv/conv.tex [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+\item[stack] ???
```
Comment (lamblin) [inline conv/conv.tex]: We used that to denote the set of input channels.

File: conv/conv.tex [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+}
+\only<2->{
+\begin{displaymath}
+C_{bo}(i, j) = \sum^{k=0}_{l-1} \sum^{x = 0}_{m-1} \sum^{y = 0}_{n-1} A_{bk}(x + i - \frac{m}{2}, y + j - \frac{n}{2}) F_{ko}(x, y)
+\end{displaymath}
+\begin{itemize}
+\item<3-> $k$ is the input channel.
+\item<4-> $o$ is the output channel.
+\item<5-> $l$ is the number of input channels.
+\item<6-> $(m, n)$ is the size of the filters.
```
Comment (lamblin) [inline conv/conv.tex]: Where does "m/2" and "n/2" come from? I think this mostly makes sense for "same" convolution, or convolutions where coordinates can be negative, but that is not the case for us.

Maybe you can start with only one input channel and one output channel, which is the regular 2D convolution (in signal processing).

**Comment by abergeron** [comment]
I've updated the drawings and explanations, definitions still remains to do.

**Comment by abergeron** [comment]
I've updated the vocabulary, everything should be ok now.

**Pull request merged**